base/linalg/triangular.jl

```diff
@@ -1900,14 +1907,10 @@ end
 function sqrtm{T}(A::UnitUpperTriangular{T})
```

Comment:
(It may be that the effect on the `UpperTriangular` case is too small to measure, however.  Or maybe even LLVM is smart enough to figure out that `i ≤ j` is always true.  But it is probably still a good idea to unwrap the type.)